website/pages/en/new-chain-integration.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Integrating New Networks
+---
+
+Support for new on-chain data sources can be added to Graph Node, allowing developers to build and deploy subgraphs and substreams-based subgraphs for new chains. Two different interfaces can be used for blockchain data ingestion through Graph Node: EVM JSON-RPC and Firehose.
+
+**1. EVM JSON-RPC**
+
+If the blockchain is EVM equivalent and the client/node exposes the standard EVM JSON-RPC API, Graph Node should be able to index the new chain. For more information on testing, refer to [Integrating an EVM equivalent chain](new-chain-integration#integrating-an-evm-equivalent-chain).
+
+**2. Firehose**
+
+For non-EVM-based chains, Graph Node will need to ingest blockchain data via gRPC and known type definitions. This can be done via [Firehose](firehose/README/), a new technology developed by [StreamingFast](https://www.streamingfast.io/) that provides a highly-scalable indexing blockchain solution using a files-based and streaming-first approach.
+
+## Difference between EVM JSON-RPC & Firehose
+
+While the two are suitable for subgraphs, a Firehose is always required for developers wanting to build with [Substreams](substreams/), like building substreams-based subgraphs. In addition, Firehose allows for improved indexing speeds when compared to JSON-RPC.
```

Comment:
I think we capitalize "Substreams-powered subgraphs"?

Reply:
I also think this raises a lot of questions about the overall architecture which aren't necessarily obvious if you don't already know about substreams, firehoses and subgraphs

Reply:
hm, got any proposal, @azf20 ? I can either remove it, or try to add more references (I don't think I had any for SbS back then).